apps/src/templates/progress/ProgressLesson.jsx

```diff
@@ -74,10 +76,18 @@ const ProgressLesson = React.createClass({
 
   getInitialState() {
     return {
-      collapsed: false
+      collapsed: this.props.currentStageId !== this.props.lesson.id
     };
   },
 
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.currentStageId !== this.props.currentStageId) {
+      this.setState({
```

Comment:
Interesting... coverage suggests we test this behavior but not its negative case: A props change that does not change the current stage should not change the collapsed state.  Worth covering?